maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,273 @@
+                        span.add_error({"message": str(e)})
+                        span.end()
+                    _agent_spans.clear()
+
+                trace = _global_maxim_trace.get()
+                if trace is not None:
+                    print("[MaximSDK] Cleaning up trace due to error")
+                    trace.add_error({"message": str(e)})
+                    trace.end()
+                    _global_maxim_trace.set(None)
+
+                raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)